kf_hungarian_tracker/kf_hungarian_tracker/kf_hungarian_node.py

```diff
@@ -177,49 +200,55 @@ def callback(self, msg):
             # add current active obstacles
             for obs in filtered_obstacle_list:
                 obstacle_uuid = uuid.UUID(bytes=bytes(obs.msg.uuid.uuid))
-                (r, g, b) = colorsys.hsv_to_rgb(obstacle_uuid.int % 360 / 360., 1., 1.) # encode id with rgb color
-                # make a cube 
+                (r, g, b) = colorsys.hsv_to_rgb(
+                    obstacle_uuid.int % 360 / 360.0, 1.0, 1.0
+                )  # encode id with rgb color
+
+                # make a cube
                 marker = Marker()
                 marker.header = msg.header
                 marker.ns = str(obstacle_uuid)
                 marker.id = 0
-                marker.type = 1 # CUBE
+                marker.type = 1  # CUBE
                 marker.action = 0
                 marker.color.a = 0.5
                 marker.color.r = r
                 marker.color.g = g
                 marker.color.b = b
                 marker.pose.position = obs.msg.position
                 angle = np.arctan2(obs.msg.velocity.y, obs.msg.velocity.x)
-                marker.pose.orientation.z = np.float(np.sin(angle / 2))
```

Comment:
Why remove the orientation?

Reply:
fixed in 6a7ebcf7a741774daf8a7306cc4b31b58aaf7412